tests/store/filters/test_status_samples_filters.py

```diff
@@ -690,3 +695,62 @@ def test_filter_get_samples_by_internal_id_pattern(
 
     # THEN the sample should have the correct name
     assert samples[0].internal_id == StoreConftestFixture.INTERNAL_ID_SAMPLE_WITH_ATTRIBUTES.value
+
+
+def test_filter_samples_by_identifier_name_and_value(
```

Comment:
I'm not sure that this tests the intended use of the function. Here we test that for each attribute/identifier we find one sample, being the one we started with. The function is supposed to find samples where the identifier chosen is equal to the given value. So what we should test is basically, do the samples we get back have the correct value for the correct attribute.